packages/relay/tests/lib/hbarLimiter.spec.ts

```diff
@@ -38,12 +40,12 @@ describe('HBAR Rate Limiter', async function () {
     });
 
     it('should be disabled, if we pass invalid total', async function () {
-        rateLimiter = new HbarLimit(logger, currentDateNow, invalidTotal, validDuration);
+        rateLimiter = new HbarLimit(logger, currentDateNow, invalidTotal, validDuration, registry);
 
         const isEnabled = rateLimiter.isEnabled();
         const limiterResetTime = rateLimiter.getResetTime();
         const limiterRemainingBudget = rateLimiter.getRemainingBudget();
-        const shouldRateLimit = rateLimiter.shouldLimit(currentDateNow);
+        const shouldRateLimit = rateLimiter.shouldLimit(currentDateNow, 'mode', 'methodName');
```

Comment:
nit: use valid mode and method values in the test cases. We'll never pass 'mode' or 'methodName'.
Makes it less confusing for someone new reading and wondering when we'd pass those values